lib/solidus_starter_frontend/engine.rb

```diff
@@ -9,6 +9,10 @@ class Engine < Rails::Engine
 
     engine_name 'solidus_starter_frontend'
 
+    initializer 'solidus_starter_frontend', before: 'solidus_support_frontend_paths' do
```

Comment:
Do we need to add `solidus_support_frontend_paths` [here](https://github.com/solidusio/solidus_support/blob/09bfde4e67496167aca79cf23f05c7d8333031ac/lib/solidus_support/engine_extensions.rb)?

Reply:
Yes, for not, it's added on my fork https://github.com/waiting-for-dev/solidus_support/blob/bcda06b7ff436018b5a3eb2588ca55a7128a7d5a/lib/solidus_support/engine_extensions.rb#L85. For now, you need to manually add it to the Gemfile if you want to test this branch.